PWGEM/PhotonMeson/Utils/PCMUtilities.h

```diff
@@ -104,4 +104,80 @@ inline void Vtx_recalculation(o2::base::Propagator* prop, T1 lTrackPos, T2 lTrac
 
   Vtx_recalculationParCov<TrackPrecision>(prop, trackPosInformation, trackNegInformation, xyz, matCorr);
 }
+//_______________________________________________________________________
+template <typename TV0>
+float getPtResolution(TV0 const& v0)
+{
+  float px = v0.px();
+  float py = v0.py();
+  float pt = RecoDecay::sqrtSumOfSquares(px, py);
+  float px_err = std::sqrt(fabs(v0.sigmaPx2()));
+  float py_err = std::sqrt(fabs(v0.sigmaPy2()));
+  float pxy_err = v0.sigmaPxPy();
+  return std::sqrt(std::pow(px / pt * px_err, 2) + std::pow(py / pt * py_err, 2) + 2.f * px / pt * py / pt * pxy_err);
+}
+//_______________________________________________________________________
+template <typename TV0>
+float getPhiResolution(TV0 const& v0)
+{
+  float px = v0.px();
+  float py = v0.py();
+  float pt = RecoDecay::sqrtSumOfSquares(px, py);
+  float px_err = std::sqrt(fabs(v0.sigmaPx2()));
+  float py_err = std::sqrt(fabs(v0.sigmaPy2()));
+  float pxy_err = v0.sigmaPxPy();
+  return std::sqrt(std::pow(px / pt / pt * py_err, 2) + std::pow(py / pt / pt * px_err, 2) - 2.f * px / pt / pt * py / pt / pt * pxy_err);
+}
+//_______________________________________________________________________
+template <typename TV0>
+float getThetaResolution(TV0 const& v0)
+{
+  float px = v0.px();
+  float py = v0.py();
+  float pz = v0.pz();
+  float pt = RecoDecay::sqrtSumOfSquares(px, py);
+  float p = RecoDecay::sqrtSumOfSquares(px, py, pz);
```

Comment:
Doesn't `v0` have `p()` and `pt()` getters? If not, it might be worth implementing them.

Reply:
Hi @vkucera , Indeed, we have them. Thank you.